cscs-checks/apps/paraview/paraview_check.py

```diff
@@ -11,8 +11,13 @@
 class ParaViewCheck(rfm.RunOnlyRegressionTest):
     def __init__(self):
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'eiger:mc']
-        self.valid_prog_environs = ['builtin']
+                              'eiger:mc', 'pilatus:mc']
+
+        if self.current_system.name in ['eiger', 'pilatus']:
+            self.valid_prog_environs = ['cpeCray']
```

Comment:
How do we use `builtin` on alps ? 
- on xc: self.valid_prog_environs = builtin lets the modulefile loads the PrgEnv-xxx (no action needed in rfm),
- on ex: should be the same ?

Reply:
@jgphpc, I do not understand you comment. 
If one selects the `builtin` on XC, then no PrgEnv-xxx is loaded. For instance, reframe only emits this for the GROMACS builtin test
```
module load daint-gpu                                                              
module load GROMACS
```

Reply:
Ah! Is it related to the hierarchical naming scheme?

Reply:
I can't check on the system but 
https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/g/GROMACS/GROMACS-2020.5-cpeGNU-21.04.eb#L23
means that it will load PrgEnv-gnu ?

Reply:
Yes, `module load GROMACS` does load its dependencies... So, it should at the end of the day, load `CrayGNU`, which will load `PrgEnv-gnu`.

Reply:
Unless, the GROMACS module is hidden behind the hierarchical naming scheme, then the module `GROMACS` won't be found. In this case, one needs to first load `cpeGNU`, then load GROMACS.

Reply:
Toochain modules must be loaded first, due to the HMNS: that applies to every application built with a toolchain.